Documentation/devel/architecture.md

```diff
@@ -46,7 +46,7 @@ where:
 - `stage1` is a copy of the stage1 ACI that is safe for read/write
 - `stage1/manifest` is the manifest of the stage1 ACI
 - `stage1/rootfs` is the rootfs of the stage1 ACI
-- `stage1/rootfs/init` is the actual stage1 binary to be executed (this path may vary according to the `coreos.com/rkt/stage1/init` Annotation of the stage1 ACI)
+- `stage1/rootfs/init` is the actual stage1 binary to be executed (this path may vary according to the `coreos.com/rkt/stage1/run` Annotation of the stage1 ACI)
```

Comment:
We have lots of undocumented files there. We don't need to document them all in this PR though.